Unity-Technologies/ScriptableRenderPipeline
# Added documentation for mip debug rendering

Small shader fix to support up to 16K textures (14 mips) rather than 4K (12 mips) in the debug rendering.
(Also changed to display red for larger textures rather than white)

### Purpose of this PR
Added documentation for mip debug rendering

---
### Release Notes
Added documentation for mip debug rendering
Updated mip debug rendering to support up to 16K textures (previous limit 4K textures)

---
### Testing status
**Katana Tests**: First off we need to make sure the Katana SRP tests are green?

**Manual Tests**: What did you do?
I enabled texture streaming and turned on the debug mode to check the rendering when writing the documentation
I used this test (after adding HDRP to the project and clicking the Set SRP button on the grid node to switch rendering shaders).
https://github.cds.internal.unity3d.com/unity/com.unity.texture.streaming

**Automated Tests**: 
Nothing added. Main focus was documentation.

Ran on Yamato:
https://yamato.prd.cds.internal.unity3d.com/jobs/78-ScriptableRenderPipeline


---
### Overall Product Risks
**Technical Risk**: Low
**Halo Effect**: Low

---
### Comments to reviewers
Notes for the reviewers you have assigned.
